.travis.yml

```diff
@@ -16,12 +16,17 @@ before_install:
   - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{,.disabled} || echo "xdebug not available"
 
 install:
-  - travis_retry composer -n --prefer-source install
+  - travis_retry composer update --no-interaction --prefer-dist  ${COMPOSER_FLAGS}
```

Comment:
Shouldn't this be done for the prefer-lowest step and only that one? I'm all for not versioning `composer.lock`, but this kind of change should happen on all branches, otherwise it's confusing for contributors.

Reply:
Without a committed lock file, `composer update` and `composer install` are the same. The reason why we're using `update` here is so we can pass the `--prefer-lowest` option below without duplicating the entire command. `install` (understandably) doesn't support `--prefer-lowest`.

Reply:
With this change applied, we're testing latest dependencies in all stages and include one stage that specifically tests against the lowest set of dependencies.

Reply:
It's great, but it's only on `master`, that's what I'm complaining about here.

Reply:
I made a bit of a mess, but the change is now in 2.13.x and master. Wanted to put it up for review, then realised I don't own a fork of this.

Reply:
I just wondered why 2.13.x looks similar to master. :-D